packages/edit-navigation/src/components/menu-editor/navigation-structure-panel.js

```diff
@@ -6,20 +6,25 @@ import { Panel, PanelBody } from '@wordpress/components';
 import { useDispatch, useSelect } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 
-export default function NavigationStructurePanel( { blocks, initialOpen } ) {
+export default function NavigationStructurePanel( {
+	menuId,
+	blocks,
+	initialOpen,
+} ) {
 	const selectedBlockClientIds = useSelect(
 		( select ) => select( 'core/block-editor' ).getSelectedBlockClientIds(),
 		[]
 	);
 	const { selectBlock } = useDispatch( 'core/block-editor' );
+	const showNavigationStructure = !! menuId && !! blocks.length;
 
 	return (
 		<Panel className="edit-navigation-menu-editor__navigation-structure-panel">
 			<PanelBody
 				title={ __( 'Navigation structure' ) }
 				initialOpen={ initialOpen }
 			>
-				{ !! blocks.length && (
+				{ showNavigationStructure && (
```

Comment:
Same as above - should this component just return null when `showNavigationStructure === false`?

Reply:
I changed this back to how it is now for the same reason as above, the panel will show at the same time as the create panel.